src/components/style-selector/simple-style-selector.vue

```diff
@@ -3,44 +3,39 @@ import { useTranslation } from 'i18next-vue'
 import { ref, watch } from 'vue'
 
 import { useStyleStore } from '@/stores/style.store'
-import { statePersistorStyleService } from '@/services/state-persistor/state-persistor-bgstyle.service'
 import SimpleStyleItem from './simple-style-item.vue'
 import { bgConfigFixture } from '@/__fixtures__/background.config.fixture'
-import { SimpleRoadStyle } from '@/composables/mvt-styles/mvt-styles.model'
+import { SimpleStyle } from '@/composables/mvt-styles/mvt-styles.model'
 import { storeToRefs } from 'pinia'
 import useMvtStyles from '@/composables/mvt-styles/mvt-styles.composable'
 
 const { t } = useTranslation()
 const styleStore = useStyleStore()
 const styleService = useMvtStyles()
 const { bgStyle } = storeToRefs(styleStore)
-const simpleStyles = ref(bgConfigFixture().simple_styles)
-
-statePersistorStyleService.bootstrapStyle()
+const simpleStyles = ref(bgConfigFixture().simple_styles.road)
 
 const props = defineProps({
   isOpen: {
     type: Boolean,
     default: true,
   },
 })
-const simpleStyleConf = bgConfigFixture().simple_styles
+const simpleStyleConf = bgConfigFixture().simple_styles.road
```

Comment:
Maybe simplify with the line above?
```suggestion
const simpleStyleConf = bgConfigFixture().simple_styles.road
const simpleStyles = ref(simpleStyleConf)
```

Reply:
ok